deploy/docker/notebook/base/base.yaml

```diff
@@ -429,67 +429,56 @@ addfiles:
           
           #### Hook Class ####
 
-          class hookVariables():
-              POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
-              work_path = '/home/jovyan/work/'
-              shared_path = '/home/jovyan/shared/notebooks'
-
           class HookFileContentsManager(FileContentsManager):
-              """Local spawner that runs single-user servers as the same user as the Hub itself.
-              Overrides user-specific env setup with no-ops.
+              """ Jupyter Server Hook Class for database synchronization with Notebooks Hub
               """
 
+              POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
+              work_path = '/home/jovyan/work/'
+              shared_path = '/home/jovyan/shared/notebooks'
+              
               def is_inside_path(self, path_to_test, path):
                   return os.path.samefile(os.path.commonpath([path, path_to_test]), path)
               
               def script_post_delete(self, os_path):
 
-                  hookVar = hookVariables()
-
                   os_path = str(Path().resolve()) + os_path
                   
-                  if not (self.is_inside_path(os_path, hookVar.work_path) or self.is_inside_path(os_path, hookVar.shared_path)):
+                  if not (self.is_inside_path(os_path, self.work_path) or self.is_inside_path(os_path, self.shared_path)):
                       return
                   
                   try:
-                      r = requests.post(hookVar.POLUS_NOTEBOOKS_HUB_API+"/file-data/delete-file", json={"user": os.getenv("JUPYTERHUB_USER"), "filePath": os_path}, timeout=1)
+                      r = requests.post(self.POLUS_NOTEBOOKS_HUB_API+"/file-data/delete-file", json={"user": os.getenv("JUPYTERHUB_USER"), "filePath": os_path}, timeout=1)
                   except Exception as e:
-                      if os.environ["POLUS_NOTEBOOKS_HUB_FILE_LOGGING_ENABLED"] == "True":
-                          logging.getLogger().error(f"Failed to delete {os_path} from Notebooks Hub", exc_info=e)
+                      logging.getLogger().error(f"Failed to delete {os_path} from Notebooks Hub", exc_info=e)
 
               def script_post_save(self, os_path):
                   
-                  hookVar = hookVariables()
-
                   os_path = str(Path().resolve()) + os_path
 
                   if "jovyan" not in os_path:
                       return
```

Comment:
Please add the comments explaining why do we need to manipulate the path here (I remember you described that in an email)